disable switch to import mode for global sort, and add tikv-worker-url config (#20866)
pingcap/docs: tidb-configuration-file.md

+ Default value: `-1`
+ When the remaining available storage in `tmp-storage-path` is lower than the value defined by `tmp-storage-quota`, the TiDB server reports an error when it is started, and exits.

> ### `tikv-worker-url` <span class="version-mark">New in v9.0.0-xxxx</span>

Reviewer: ```suggestion
### `tikv-worker-url` <span class="version-mark">New in v9.0.0</span>
```

> + Specifies the url of tikv-worker. This option is added but currently unused and may be removed in a future release without notice.

Reviewer: ```suggestion

> **Warning:**
>
> This configuration item is currently unused and might be removed in a future release without prior notice.

+ Specifies the URL of `tikv-worker`.
```

+ Default value: `-1`
+ When the remaining available storage in `tmp-storage-path` is lower than the value defined by `tmp-storage-quota`, the TiDB server reports an error when it is started, and exits.

### `tikv-worker-url` <span class="version-mark">New in v9.0.0</span>

> 

Reviewer: ```suggestion
```
